biogeochem/CMIP data.txt

```diff
@@ -0,0 +1,24 @@
+. 	
```

Comment:
@ZacharyRobbins @ckoven @jkshuman should this file be retained?

Reply:
it should be deleted.